Simple Incremental/Assets/Scripts/CharacterHealthEffect.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[RequireComponent(typeof(CharacterHealth))]
+public class CharacterHealthEffect : MonoBehaviour
+{
+    CharacterHealth characterHealth;
+    ParticleSystem system;
+
+    void Awake()
+    {
+        characterHealth = GetComponent<CharacterHealth>();
+        system = GetComponent<ParticleSystem>();
```

Comment:
If you use Particle System, maybe require it as well?

Reply:
Ok ill do that. Thanks.